spec/workflow_service_spec.rb

```diff
@@ -586,4 +587,31 @@
       expect(Dor::WorkflowService.get_lane_ids('dor', 'accessionWF', 'shelve')).to eq(%w(lane1 lane2))
     end
   end
+
+  describe '.workflow_resource_method' do
+    let(:stubs) do
+      Faraday::Adapter::Test::Stubs.new do |stub|
+        stub.get('x?complete=a&complete=b') do |env|
+          [200, {}, 'ab']
+        end
+      end
+    end
```

Comment:
OK, it seems things actually execute because of the global `before :each` at
https://github.com/sul-dlss/dor-workflow-service/blob/master/spec/workflow_service_spec.rb#L49
(No problem.)